geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseIndexAdapter.scala

```diff
@@ -67,9 +69,9 @@ class HBaseIndexAdapter(ds: HBaseDataStore) extends IndexAdapter[HBaseDataStore]
   import scala.collection.JavaConverters._
 
   override def createTable(
-      index: GeoMesaFeatureIndex[_, _],
-      partition: Option[String],
-      splits: => Seq[Array[Byte]]): Unit = {
+                            index: GeoMesaFeatureIndex[_, _],
```

Comment:
It's typically good practice to only include code changes, even styling, that are only relevant to the content of the PR. @elahrvivaz What's your opinion?

Reply:
This was probably just IntelliJ autoformatting, I can just revert it if necessary

Reply:
yeah, looks like a lot of auto-formatting changes. @alex-shen1  could you revert those? thanks.

Reply:
Should be reverted in latest commit